django/core/management/commands/sqlmigrate.py

```diff
@@ -55,8 +55,12 @@ def handle(self, *args, **options):
                 migration_name, app_label))
         targets = [(app_label, migration.name)]
 
-        # Show begin/end around output only for atomic migrations
-        self.output_transaction = migration.atomic
+        # Show begin/end around output only for atomic migrations and if the database
+        # supports transactional ddl
+        if 'can_rollback_ddl' in connection.__dict__:
+            self.output_transaction = migration.atomic and connection.can_rollback_ddl
+        else:
+            self.output_transaction = migration.atomic
 
```

Comment:
This can be simplified:
```python
self.output_transaction = migration.atomic and getattr(connection, 'can_rollback_ddl', True)
```

Reply:
Thanks for the suggestion, Do I have to do any changes from my side, I'm new here and I don't see anything like accept the suggestion.

Reply:
You need to make the changes.